test-lib.sh

```diff
@@ -1195,7 +1195,12 @@ function ct_timestamp_diff() {
 function ct_get_certificate_timestamp() {
   local container=$1
   local path=$2
-  date '+%s' --date="$(docker exec -ti "$container" bash -c "cat $path" | openssl x509  -startdate -noout | grep notBefore | sed -e 's/notBefore=//')"
+set -x
```

Comment:
It would not be harmful IMO, if the `set -x` stays here even after debugging of this PR is finished. But then would be better to append closing `set +x` at the end of this function.

Reply:
This was likely an artifact left from debugging. I'd rather not handle with `-x` within some tests. I'll remove this.